### PR TITLE
Fix Error In README.md Example Program

### PR DESCRIPTION
**Pull request by csm-18**
Add main function to the program and put the ui window code in it.

### CODE REVIEW

**Comment by ttytm** [comment]
Hey @csubrahmanyam,

for a single-v-file program the `main` function can be omitted. Just as the minimal example does.

https://github.com/webui-dev/v-webui/blob/fa43fbaae1f480bc922991acaaafd4ab6bc5d9c1/examples/minimal.v#L1-L5

> fn main() declaration can be skipped in one file programs.

<small>Ref. https://github.com/vlang/v/blob/master/doc/docs.md#hello-world</small>

**Comment by csm-18** [comment]
But this is the output I am getting when I run the above simple example.

image

**Comment by ttytm** [comment]
> But this is the output I am getting when I run the above simple example.
> 
> image

In case of a single file program specifying the filename is required.

```sh
v -cc gcc run hello.v
```

AFAIK, there isn't an idiomatic way to add or not add the `main` function in the case of a simple example; it comes down to preference. We omit it in the minimal example, and I don't have a strong preference, so we can add it :+1: .

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ v install https://github.com/webui-dev/v-webui
 ```v
 import vwebui as ui
 
+
+
+
```
Comment (ttytm) [inline README.md]: To make it into a successful contribution, can you please keep the code in the example formatted without adding additional line breaks and by adding indentation to the main function block.

Comment (csm-18) [reply README.md]: Sure!

Comment (csm-18) [reply README.md]: Did it!

Comment (hassandraga) [reply README.md]: LGTM.

Comment (ttytm) [reply README.md]: Thanks for the contribution!

**Pull request merged**